charts/rd-profile-sync/values.yaml

```diff
@@ -9,8 +9,12 @@ java:
     POSTGRES_PORT: 5432
     POSTGRES_CONNECTION_OPTIONS: "?sslmode=require"
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    IDAM_URL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
     USER_PROFILE_URL: http://rd-user-profile-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     OAUTH2_REDIRECT_URI: https://rd-professional-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/oauth2redirect
+    OIDC_ISSUER_URL: https://forgerock-am.service.core-compute-idam-{{ .Values.global.environment }}.internal:8443/openam/oauth2/hmcts
+    OPEN_ID_API_BASE_URI: https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net/o
+    image: 'hmctspublic.azurecr.io/rd/professional-api:latest'
 
   image: 'hmctspublic.azurecr.io/rd/profile-sync:latest'
   keyVaults:
```

Comment:
You need following 2 entries to get secret vault value while starting spring boot. Othrwise sprint boot will fail to load. 

You need to add following entry under secrets in valves.yml
- OPEN-ID-API-BASE-URI

Also in bootstrap.yml you will require below entry under aliases
OPEN-ID-API-BASE-URI: spring.security.oauth2.client.provider.oidc:issuer-uri